skopt/gp_opt.py

```diff
@@ -95,7 +96,9 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         Number of calls to `func`.
         If `n_random_starts` > 0, `n_calls - n_random_starts`
         additional evaluations of `func` are made that are guided
-        by the `base_estimator`.
+        by the `base_estimator`. If `x0` is provided but not `y0`
+        then `n_calls - len(x0) - n_random_starts` evaluations
+        are made instead of `n_calls - n_random_starts` .
```

Comment:
Ok, what do you think about this  : 


      * `n_calls` [int, default=100]:
          Number of calls to `func`.
          If `x0` is provided but not `y0`, then the elements of `x0` are
          first evaluated, followed by `n_random_starts` evaluations.
          Finally, `n_calls - len(x0) - n_random_starts` evaluations are
          made guided by the surrogate model. If `x0` and `y0` are both
          provided then `n_random_starts` evaluations are first made then
          `n_calls - n_random_starts` subsequent evaluations are made
          guided by the surrogate model.